G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice.Tests/Commands/OutlookSaveAttachment/ShouldSaveAttachment_WhenSaveAttachmentCommandIsCalled.G1ANT

```diff
@@ -0,0 +1,27 @@
+﻿♥outlookInboxFolder = ‴t.g1ant.test@gmail.com\Skrzynka odbiorcza‴
+♥toBeSentAttachmentPath = ‴♥environment⟦TEMP⟧\testattachmentsent.txt‴
```

Comment:
Check if these files does not exist, if they do, delete them before proceeding with the test.

Reply:
ping

Reply:
file.delete ♥toBeSentAttachmentPath timeout 2
file.delete ♥toBeDownloadedAttachmentPath timeout 2